### ISSUE TITLE
Number of days ago does not match date in death report

### ISSUE THREAD

**Issue by abbyad** [medic/cht-core#6063]
**Describe the bug**
The "days ago" in the list of deceased person does not correspond to calendar days. This may be due to a timezone issue. For example marking a date of death as yesterday at 11am EDT in the report shows as "2 days ago" when looking at the report at 3:30pm EDT.

**To Reproduce**
I am creating this issue as a placeholder without having a chance to explore fully, but here are the steps that I went through to see this behavior: 
1. At 11am EDT submit a death report with the date of death set to the previous day.
2. Wait 10+ minutes to see the patient in the list of deceased persons.
3. See that the date is 2 days ago instead of 1 day ago (seen at 3:30pm EDT, but may happen earlier).

**Expected behavior**
As the person who submitted a death report I would expect the number of "days ago" to match calendar days for my local timezone.

**Environment**
- Instance: cht.app
- Browser: Chrome 77
- App: webapp
- Version: 3.7.0-beta.3

**Additional context**
This issue was discovered while reproducing https://github.com/medic/cht-core/issues/6053. I didn't have a chance to fully explore timing and timezone issues to narrow down the problem, but wanted to create the issue as a placeholder. We have encountered similar situation in the past with https://github.com/medic/cht-core/issues/4232, so we should see if there is a common way to handle these scenarios.

**Comment by garethbowen** [comment]
I had assigned this to myself because it was somewhat blocked by #6053 . I've had trouble reproducing this in part because of my timezone but it may also be that the fix for #6053 also fixed this. Now that that code is merged into master this is ready to be picked up by anyone so I'm unassigning myself.

**Comment by wtekeu** [comment]
I initially tried to reproduce the bug without much success. It was showing the right relative date:
<img width="730" alt="Screen Shot 2019-12-04 at 12 55 42 PM" src="https://user-images.githubusercontent.com/1445164/70167848-98570780-1695-11ea-8fce-1bf5e6d79993.png">

So, I changed the code to what it was before the fix for https://github.com/medic/cht-core/issues/6053 and obtained the following which seems to be consistent with Marc's experience:
<img width="721" alt="Screen Shot 2019-12-04 at 12 54 49 PM" src="https://user-images.githubusercontent.com/1445164/70167836-90976300-1695-11ea-83ba-23a047b0aad3.png">

So it looks the fix for https://github.com/medic/cht-core/issues/6053 does fix this one as well.

**Comment by garethbowen** [comment]
@abbyad Would you mind ATing this on the latest `master` to confirm?

**Comment by abbyad** [comment]
If I can't AT it tomorrow, I'll leave a note here for someone else to pick it up since I won't be able to get to it next week.

**Comment by abbyad** [comment]
Tested on 3.6.1 to reproduce, then upgraded to master, and this issue now passes AT. Since this was resolved by the work on https://github.com/medic/cht-core/issues/6053 there is no PR to merge. 

**Here are some of the test details:**

Using 3.6.1, the following patients were marked as deceased 3, 1, 2 days ago and today, respectively, and the date shown in the deceased list was incorrect for those before today. This was done at 9pm EST, and as visible within 10 minutes of marking them as deceased.

![image](https://user-images.githubusercontent.com/789512/70367130-2801df00-186b-11ea-9539-6c7f85a92066.png)

The same instance was upgraded to [master ~3.8](https://github.com/medic/cht-core/tree/2a238abb32624b9111bec9a81f57f54cddc64d54) and the same people were immediately showing the correct dates:
![image](https://user-images.githubusercontent.com/789512/70367262-5e8c2980-186c-11ea-96c9-94f3ff23d1e5.png)

I recreated the initial steps with another family starting on master, and it also displayed the dates correctly:
![image](https://user-images.githubusercontent.com/789512/70367503-841a3280-186e-11ea-89f9-2e2761a04467.png)
